rasa/core/channels/twilio_voice.py

```diff
@@ -0,0 +1,334 @@
+import re
+from sanic import Blueprint, response
+from sanic.request import Request
+from sanic.response import HTTPResponse
+from twilio.twiml.voice_response import VoiceResponse, Gather
+from typing import Text, Callable, Awaitable, List, Any, Dict, Optional
+
+from rasa.utils.io import get_emoji_regex
+from rasa.shared.utils.io import raise_warning
```

Comment:
please don't import functions directly but rather reference them by their module (see [here](https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#70d0247fdd1f48d2aec341fd91fe1dac))
```suggestion
import rasa.utils.io
import rasa.shared.utils.io
```